webpack.config.babel.js

```diff
@@ -160,7 +160,7 @@ const basePlugins = (bundle_name) => ([
     // Increment BASE_32_VERSION with each release following Base32 notation, i.e AA -> AB
     // Do it only when we introduce a breaking change between SDK and cross device client
     // ref: https://en.wikipedia.org/wiki/Base32
-    'BASE_32_VERSION' : 'AM',
+    'BASE_32_VERSION' : 'AN',
```

Comment:
Outside of the scope of this PR, and maybe you don't want to do it anyway - but just fyi

You could keep this version number in decimal, and let JS do the number conversions for you. I.e. `(343).toString(32).toUpperCase()` rather than `'AN'`

Otherwise people are gonna get super confused when they see `AV` --> `B0`, whereas `(351).toString(32).toUpperCase()` --> `(352).toString(32).toUpperCase()` makes sense

Reply:
Oh, good point!

Reply:
We can change this as part of the release automation

Reply:
https://jira.onfido.co.uk/browse/CX-2809 :)